src/plugins/data/common/search/session/types.ts

```diff
@@ -31,6 +31,13 @@ export interface SearchSessionSavedObjectAttributes {
    * Expiration time of the session. Expiration itself is managed by Elasticsearch.
    */
   expires: string;
+  /**
+   * Time of transition into completed state,
+   *
+   * Can be "null" in case already completed session
+   * transitioned into in-progress session
+   */
+  completed?: string | null;
```

Comment:
@lizozom, @lukasolson I had to add `completed` time into a search session. I guess this is what we want. 

Who we would like to handle sessions created and completed in older versions: 

1. We could add a migration that would put `touched` -> `completed` for session in completed status
2. We could skip adding the migration. In this case, we should have in mind that there is a possibility of a completed session with an empty `completed` time field. 
2.1 In this case we can show `touched` time in UI
2.2 We can just hide `completed` in this case in UI

Reply:
If the status is complete, then is the value for `touched` always going to be the same as `completed`? If so, is it necessary to add another field, or can we just use the `touched` value (for completed sessions)?

Reply:
> If the status is complete, then is the value for touched always going to be the same as completed? 

This is not currently the case :( 
`touched` is update on any `update()` call. 
So for example - changing the session name via `update()` would bump `touched`

Reply:
I added a migration